server/src/test/java/org/opensearch/cluster/coordination/NoMasterBlockServiceTests.java

```diff
@@ -61,22 +61,22 @@ private void assertDeprecatedWarningEmitted() {
 
     public void testBlocksWritesByDefault() {
         createService(Settings.EMPTY);
-        assertThat(noMasterBlockService.getNoMasterBlock(), sameInstance(NO_MASTER_BLOCK_WRITES));
+        assertThat(noClusterManagerBlockService.getNoMasterBlock(), sameInstance(NO_MASTER_BLOCK_WRITES));
```

Comment:
`NO_MASTER_BLOCK_WRITES` -> `NO_CLUSTER_MANAGER_BLOCK_WRITES`

Reply:
Thanks for the reminding! I found `NO_MASTER_BLOCK_WRITES` is a public variable, so I plan to change it in the next phase, and targeted for 3.0 version.